### PR TITLE
Update manage-hybrid-exchange-recipients-with-management-tools.md

### PR DESCRIPTION
**Pull request by cunninghamp**
Please make this important warning more visible by including it at the top of the document as well.

### CODE REVIEW

File: Exchange/ExchangeHybrid/manage-hybrid-exchange-recipients-with-management-tools.md [hunk 1]
```diff
@@ -21,7 +21,12 @@ manager: serdars
 
 In Exchange Hybrid environments, you must have an active Exchange Server to manage recipients attributes. First, attributes are edited using an Exchange Server in on-premises Active Directory (AD), that are then copied to Azure AD using directory synchronization. On-premises recipients can’t be modified directly in Azure Active Directory (Azure AD) or Exchange Online. Therefore you have to keep an Exchange Server running using directory synchronization via Azure Active Directory (AAD) Connect tool.
 
-If you keep an Exchange server running just for recipient management, you may be able to shut down your last Exchange server and manage recipients using Windows PowerShell. 
+If you keep an Exchange server running just for recipient management, you may be able to shut down your last Exchange server and manage recipients using Windows PowerShell.
+
+### Important: Be Aware
+
+>[!Warning]
+>Once you shut down your last Exchange server and perform the Exchange hybrid and Active Directory cleanup steps listed below, you should **erase and reformat your last Exchange server**. **Do Not Uninstall the Exchange Server**
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/manage-hybrid-exchange-recipients-with-management-tools.md:29]:
```suggestion
### Important: Be aware

> [!WARNING]
> Once you shut down your last Exchange server and perform the Exchange hybrid and Active Directory cleanup steps listed below, you should **erase and reformat your last Exchange server. _Do not uninstall the Exchange server._**
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @cunninghamp, kindly commit the suggested changes. Many thanks.

**Comment by cunninghamp** [comment]
> 

@JohanFreelancer9 I copied Microsoft's own formatting from elsewhere on the page so its really up to them whether they want to make those changes.

**Comment by cunninghamp** [comment]
A different warning has been added to an appropriate place on the page by MSFT making this PR redundant.

**Pull request closed without merging**